src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --active-link-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-size: 12px;
+  text-transform: uppercase;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list :not(:first-child) {
```

Comment:
```suggestion
.nav_list:not(:first-child) {
```

Reply:
When I delete space between nav_list and :not(:first-child), margins between links disappear.